pkgs/applications/science/programming/scyther/cli.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, cmake, glibc, makeWrapper, fetchFromGitHub, flex, bison, source }:
+stdenv.mkDerivation { 
+  name = "scyther-cli-${source.rev}";
+  src = source;
+  preConfigure = ''
+    cd src
+    # Since we're not in a git dir, the normal command this project uses to create this file wouldn't work
+    printf "#define TAGVERSION \"${source.rev}\"\n" >> version.h
```

Comment:
The Python script writes to the file, you append to it. Your use of printf is potentially wrong depending on the value of ${source.rev}, so please use a format string.